### PR TITLE
story/FLCRM-9629 Expose record links table

### PR DESCRIPTION
**Pull request by toddrun**
**Proposed Changes:**

Makes the "record_links" accessible via the Query API. The tables referenced in this PR were applied to the database in https://fulcrumapp.atlassian.net/browse/FLCRM-9606 and https://fulcrumapp.atlassian.net/browse/FLCRM-9784. 

**Testing Steps:**

To test this:

- Find the `version` in the `publish-npm` step of the Tekton pipeline run for this PR
- Set the version as the value of the `@fulcrumapp/fulcrum-schema` dependency in the package.json of the schema-service
- Skaffold the schema-service and wait for it to come up
- Assuming https://github.com/fulcrumapp/query/pull/196 has been deployed to Query, you should now be able to run these queries (via Postman or curl):
- `select * from tables` - you should see "record_links" in the results
- `select * from record_links` - you should at least see the fields returned, rather than an error

I performed this in my namespace (toddrun) instead of the preview env, but was able to query for link data:
Screen Shot 2022-08-15 at 1 02 09 PM

### CODE REVIEW

File: script/extract-schema.js [hunk 1]
```diff
@@ -62,3 +62,4 @@ convertTable('Signatures', schema.systemSignaturesTable, 'signatures', 'signatur
 // convertTable('QueryRecords', schema.systemFormTableColumns, 'query-records', schema.systemFormViewColumns);
 // convertTable('QueryChildRecords', schema.systemRepeatableTableColumns, 'query-child-records');
 // convertTable('QueryValues', schema.systemValuesTableColumns, 'query-values');
+convertTable('RecordLinks', schema.systemSignaturesTable, 'signatures', 'signatures', schema.organizationViews.signatures);
```
Comment (chrisworkmode) [inline script/extract-schema.js]: I believe this should reference the new stuff you added for record links instead of signatures.

**Review by chrisworkmode — DISMISSED** [review]

File: script/extract-schema.js (hunk 1 above)
Comment (chrisworkmode) [reply script/extract-schema.js]: TBH though, I'm not even sure where this script is used.

Comment (zhm) [reply script/extract-schema.js]: I think this script can be removed now. It was used to generate the very first set of the files from the definition, so it was a one time script and not necessary anymore. 💣 it in a separate pr?

Comment (chrisworkmode) [reply script/extract-schema.js]: If it's just deleting this one file, @toddrun could probably 💣 it here.

**Review by chrisworkmode — DISMISSED** [review]

File: script/extract-schema.js (hunk 1 above)
Comment (toddrun) [reply script/extract-schema.js]: Thanks for catching this! It's gone

**Review by osbornk — APPROVED** [review]

**Review by chrisworkmode — APPROVED** [review]

**Pull request merged**